lib/blog-feed-loader.js

```diff
@@ -0,0 +1,65 @@
+var sanitizeHtml = require('sanitize-html');
```

Comment:
Er, on the other hand, we *are* retrieving the HTML from Google, which is retrieving it from https://blog.webmaker.org, so as long as we trust those, we can trust the HTML too. I do like the idea of sanitizing the HTML for extra security, to ensure that compromising the blog doesn't also compromise teach, but the extra weight of a client-side HTML sanitizer... :sweat: